spec/PropertyBuilder/AttributeBuilderSpec.php

```diff
@@ -22,7 +22,7 @@
 
 final class AttributeBuilderSpec extends ObjectBehavior
```

Comment:
The production class has been changed, but there is no change in spec file. Could you check all classes you changed by this?

Reply:
It still pops up

Reply:
the changed part of the code is not reflected in the spec tests